tests/eas/load_tracking.py

```diff
@@ -537,6 +540,371 @@ def test_load_avg_behaviour(self, experiment, tasks):
         """
         return self._test_behaviour(experiment, tasks, 'load_avg')
 
+class _CPUMigrationBase(LisaTest):
+    """Base class for shared functionality of tasks migration tests"""
+    test_conf = {
+        'tools'    : [ 'rt-app' ],
+        'ftrace' : {
+            'events' : [
+                'sched_switch',
+                'sched_load_cfs_rq',
+            ],
+        },
+        'modules': ['cpufreq', 'cgroups'],
+    }
+
+    # Allowed error margin
+    allowed_util_margin = UTIL_SCALE * 0.02
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(_CPUMigrationBase, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def get_wload(cls, test_env, tasks, workload_name):
+        """Create a specification for a rt-app workload from a list of tasks
```

Comment:
So this is super nitpicky of me and I feel ashamed of asking you to change this but at the same time I feel like I need to say it...

The 2 styles of docstring I've seen in LISA are
```
"""hello world"""
```
and
```
"""
hello world
"""
```
So I think it would be nice if your docstrings fit in either of those 2 categories. Sorry for having an easily-triggered OCD :(

Reply:
Done